course_access_groups/models.py

```diff
@@ -9,9 +9,10 @@
 from django.db import models
 from model_utils import models as utils_models
 from organizations.models import Organization
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 
 
-class CourseAccessGroup(models.Model):
+class Group(utils_models.TimeStampedModel):
```

Comment:
Why 'Group' instead of 'CourseAccessGroup'?

Reply:
Because `course_access_groups.models.CourseAccessGroup` sounds too verbose, what do you think?

Reply:
Renamed back to the verbose form.